requirements.bazel.txt

```diff
@@ -36,7 +36,8 @@
 absl-py
 certifi
 chardet
-gevent
+# Note: this is a temporary for local builds until CI is running py3.9+
+gevent~=25.8
```

Comment:
This and requirements.bazel.lock shouldn't be needed. I suggested this to Craig as a workaround to fix local bazel build for py 3.13. I believe Craig meant to revert it.